src/js/utilities/postHandling.mjs

```diff
@@ -0,0 +1,231 @@
+const API_BASE_URL = `https://api.noroff.dev/api/v1`;
+
+async function retrieveApiPostData(url) {
+  try {
+    // Retriving the key that has allready been stored from login
+    const token = localStorage.getItem("accessToken");
+    if (!token) {
+      console.log("No access token available.");
```

Comment:
Rather throw an error here, and where you catch it, display a message to the user.